src/interest-groups/utils.js

```diff
@@ -0,0 +1,8 @@
+import {
+	validateParam,
+} from '../utils/';
+
+export const getInterestGroup = (owner, name) => {
```

Comment:
the name of this function starts with `get` but it doesn't return anything. Perhaps `validateInterestGroup` is a better name?

Reply:
this was only meant as a stub function; I removed it

commit: dfb53cffc6be2f54d7f48ed878ad1829e985b970